src/themes/money/theme.json

```diff
@@ -815,6 +815,25 @@
     }
   },
 
+  "badge" : {
+    "base": {
+      "display": "flex",
+      "justifyContent": "center",
+      "alignItems": "center",
+      "paddingX": "base",
+      "height": 32,
```

Comment:
In my opinion, height and all the flexbox stuff would be better replaced with `paddingY`. doesn't make a massive amount of difference, though!